docs/source/tutorial/step2_datasets_evaluators.rst

```diff
@@ -99,9 +99,10 @@ many redundant lines will appear in your console.
 Therefore, it is convenient to register these extensions
 only at workers of rank zero as follows::
 
+  log_interval = chainermn.get_epoch_trigger(1, train, args.batchsize, comm)
```

Comment:
No. chainermn.get_epoch_trigger() is not used in this if block because this if block is performed by only one worker which has rank 0.